src/options/options.js

```diff
@@ -31,6 +33,26 @@ Array.prototype.forEach.call(elements.licenseInputs, (element) => {
   });
 });
 
+function initEnableSearchStorageButton() {
```

Comment:
By the logic of this function (`initEnableSearchStorageButton()`), when we will open the options page for the very first time (after the install), the checkbox `enableSearchStorageCheckbox` should be checked because if `enableSearchStorage` is not present in the storage, it is `elements.enableSearchStorageCheckbox.checked` is set to true.

But, this is not happening. Try adding some console.log statements in this function and you will see that `enableSearchStorageOption` is undefined and `elements.enableSearchStorageCheckbox.checked` is false.

Reply:
It is working fine, I have tried that many times.

`initEnableSearchStorageButton()` is just to check what setting the user had saved last time, and check the box accordingly.

Reply:
```
chrome.storage.sync.get(['enableSearchStorage'], (res) => {
    if (chrome.runtime.error) enableSearchStorageOption = true;
    else enableSearchStorageOption = res.enableSearchStorage;
    
    elements.enableSearchStorageCheckbox.checked = enableSearchStorageOption;
});
```
By reading these lines I am getting the intuition that if the `enableSearchStorage` key is not present in the extension storage, then `enableSearchStorageOption` variable will be set to true.
and hence `elements.enableSearchStorageCheckbox.checked` will be set true. 
But, this is not happening. 
- One indication is that the checkbox is not true out of the box (which it should be, as we want the feature to be enabled out of the box).
- Also, as I have stated before, you can add some console.log() statements in the function. Checkout the following GIF where I have done that.
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/34679965/73846983-702d8580-484b-11ea-87fa-37aadd90c228.gif)

The above happens till the user have not themselves manually not clicked the checkbox.

Reply:
I rechecked it. Here is what I found.

![log](https://user-images.githubusercontent.com/54908728/73874399-76d2f180-4879-11ea-8d20-81a06f28ec68.gif)

Reply:
Here is how the code works

![main](https://user-images.githubusercontent.com/54908728/73874465-91a56600-4879-11ea-8dd7-9432ac2276a3.gif)

Reply:
@AmanR28 Yeah, but my point is completely different. Do the following and paste the GIF below.
- Remove the extension from chrome.
- Load the extension again.
- Go to the options page and **Do not click the checkbox yet**.
- Open the console.

What do you see?

Reply:
@makkoncept Sorry, I didn't try that. I will take care of it also from next time.

I will fix it asap.